test/js_test.go

```diff
@@ -6099,13 +6120,13 @@ func TestJetStreamBindConsumer(t *testing.T) {
 		t.Fatalf("Unexpected error: %v", err)
 	}
 	_, err = js.SubscribeSync("foo", nats.Bind("foo", "push"))
-	if err == nil || err != nil && !strings.Contains(err.Error(), "consumer not found") {
+	if err == nil || err != nil && !errors.Is(err, nats.ErrNoConsumer) {
```

Comment:
I figure you probably wanted to make a minimal change here, but we can simplify this a little further.
```suggestion
	if err == nil || !errors.Is(err, nats.ErrNoConsumer) {
```